evals/elsuite/audio/eval.py

```diff
@@ -298,7 +298,7 @@ def get_expected(self, sample: Sample):
 class SpokenTools(MatchAudioTask):
     def load_dataset(self):
         # TODO: create samples for each user message in the row.
-        ds = load_hf_dataset(self.dataset).cast_column(
+        ds = load_hf_dataset(self.dataset, evals.eval._MAX_SAMPLES).cast_column(
```

Comment:
Not a big deal, but if you apply max_samples here and then filter later, the number will be slightly lower than expected.

Reply:
Yeah. I'll revisit this in a followup where I actually hook up the multiple tool calls per row.